isis/src/control/objs/ControlNet/ControlNet.cpp

```diff
@@ -1101,6 +873,7 @@ namespace Isis {
    */
   QList< ControlMeasure * > ControlNet::GetMeasuresInCube(QString serialNumber) {
     ValidateSerialNumber(serialNumber);
+//    return m_controlGraph[m_vertexMap[serialNumber]].measures;  <--- segfaults. 
```

Comment:
I'm concerned that we may have invalidated the vertexMap somehow. Is there anywhere else that we use the vertedMap to get a vertex descriptor by serial number and then use that descriptor?

Reply:
It'll probably take going into a debug to figure this out. Do you have a good test case where this seg faults?

Reply:
Looking over the code, we are using the vertedMap this way elsewhere, so I'm curious if the measure list is not being managed correctly.

Reply:
I can no longer reproduce the segfault problem after rebuilding everything. I'm not sure what I changed to fix this, but the problem seems to have disappeared.